feature/experimental/bgp/otg_tests/bgp_remove_private_as/README.md

```diff
@@ -0,0 +1,35 @@
+# RT-1.11: BGP remove private AS 
+
+## Summary
+
+BGP remove private AS
+
+## Procedure
+
+*   Establish BGP sessions as follows between ATE and DUT .  
+*   ATE port 1 emulates two eBGP neighbors peering the DUT. 
+*   eBGP neighbor # 1 is injecting routes with AS_PATH modified to have private AS numbers 
+*   Validate that private AS numbers are stripped before advertisement to the eBGP neighbor on ATE 
+    port 2. 
```

Comment:
There's extra whitespace at the end of these lines, and the space before the period in the first line should be removed.

Reply:
Updated.